app/src/migrations/1532427701521-add-text-index.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable import/prefer-default-export */
+// const Metadata = require('models/metadata.model');
+
+export async function up() {
```

Comment:
this is the migration placeholder. Right now, it just returns an empty promise. Basically, I spent a lot of time making this work, only to find out Mongoose can create indexes automatically if you define them in the model..... fail!

But, because I see this being useful in the future, I left it here, as reference for us-in-6-months.